common/djangoapps/course_modes/views.py

```diff
@@ -148,6 +150,21 @@ def get(self, request, course_id, error=None):
             "responsive": True,
             "nav_hidden": True,
         }
+
+        if enterprise_enabled():
+            enterprise_learner_data = get_enterprise_learner_data(user=request.user)
+            if enterprise_learner_data:
+                organizations = organization_api.get_course_organizations(course_id=course.id)
+                context["show_enterprise_context"] = True
+                context["partner_names"] = partner_short_name = course.display_organization \
+                    if course.display_organization else course.org
+                context["enterprise_name"] = enterprise_learner_data[0]['enterprise_customer']['name']
+                context["username"] = request.user.username
+                if organizations:
+                    context["partner_names"] = ','.join([
```

Comment:
I think instead of a comma, we should join the partner names using ` and ` (note space characters), so that in the display it would look like:

`PartnerX and Partner2X`, instead of `PartnerX,Partner2X`